src/main/java/TaskManager.java

```diff
@@ -0,0 +1,50 @@
+public class TaskManager {
+    private static int taskNo = 0;
+    final private static int maxTasks = 100;
+    final static Task[] tasks = new Task[maxTasks];
+
+    static void addTask(Task task) {
+        if (taskNo < maxTasks) {
+            tasks[taskNo] = task;
+            taskNo++;
+
+            System.out.print("Got it. I've added this task:\n");
+            System.out.printf("   %s\n", task);
+            System.out.printf("Now you have %d tasks in the list.\n", taskNo);
+        }
+    }
+
+    static void addTodo(String todoName) throws TodoException {
+        if (todoName.isEmpty()) {
+            throw new TodoException();
+        }
+        Todo todo = new Todo(todoName);
+        addTask(todo);
+    }
+
+    static void addDeadline(String deadlineName, String deadlineDue) {
+        Deadline deadline = new Deadline(deadlineName, deadlineDue);
+        addTask(deadline);
+    }
+
+    static void addEvent(String eventName, String eventTime) {
+        Event event = new Event(eventName, eventTime);
+        addTask(event);
+    }
+
+    static void listTasks() {
+        System.out.println("Here are the tasks in your list:");
+        for (int i = 0; i < taskNo; ++i) {
+            Task task = tasks[i];
+            System.out.printf("%d. %s\n", i + 1, task);
```

Comment:
This is not too big a deal (`printf` vs `println`), though you can consider simply sticking to `print` (not `printf`) as Java has its own handy `toString` methods that can convert `Integer` for you. I believe `printf` may be a habit from C.
```suggestion
            System.out.println((i + 1) + ". " + task);
```